COPYING

```diff
@@ -5,23 +5,41 @@ Source License, version 1.0, or at your option, any later version ("BOSL"). See
 the file ./LICENSE-BOSL for the terms of the Bootstrap Open Source Licence,
 version 1.0.
 
+In this document,
+
+- "Zcash" means the blockchain defined by the most recent agreement between
+  the Electric Coin Company and the Zcash Foundation according to section 6.2 (b)
+  of the Zcash Trademark Agreement.
+- A "Zcash Chain Fork" means a blockchain that descends from the Zcash
+  blockchain and that is or was forked within 3456 blocks of the current block
+  height of the Zcash blockchain at the time of the fork;
+- A "Zcash Technology Testnet" means a blockchain designed for the purpose of
+  testing technology potentially of use to the Zcash blockchain and not for
+  the purpose of implementing any token of economic value, where this purpose
+  and lack of such economic value are clearly communicated to its users and to
+  the general public.
+
 Only if this Original Work is included as part of the distribution of one of the
 following (each, the "Project"):
 
-- The Zcash projects published by the Electric Coin Company;
-- The Zebra project published by the Zcash Foundation;
-- A project that is designed to integrate with Zcash and provides additional
-  functionality or utility to the Zcash network and holders of the ZEC coin; or
-- A blockchain that descends from the Zcash blockchain and that is forked
-  within 100 blocks of the current block height of the Zcash blockchain at the
-  time of the code fork;
+- A project that implements Zcash;
+- A project that implements a Zcash Chain Fork;
+- A project that implements a Zcash Technology Testnet;
+- A project that is designed to integrate with Zcash (whether or not it can
```

Comment:
My intent was that if you add features for a chain fork then you should make a reasonable effort to provide them for the Zcash chain. Sometimes this might not be possible, if the features rely on particular consensus or peer-to-peer protocol differences. I'm not sure that the intent *can* be fully captured in legal wording; at some point you have to assume good faith.

Reply:
Note that this is only a clarification; it does not change the legal meaning. It's less important than the changes on lines 27 and 28.